Published package does not work

When adding `solid-panes@95-ci-publish` (i.e. the published package of the revamp branch) to mashlib, I run into the following errors:

> ERROR in ./node_modules/solid-panes/index.js
> Module not found: Error: Can't resolve './home/homePane' in '<workspace>/mashlib/node_modules/solid-panes'
>  @ ./node_modules/solid-panes/index.js 134:9-35
>  @ ./index.js
>  @ multi ./index.js
> 
> ERROR in ./node_modules/solid-panes/index.js
> Module not found: Error: Can't resolve './pad/padPane' in '<workspace>/mashlib/node_modules/solid-panes'
>  @ ./node_modules/solid-panes/index.js 53:9-33
>  @ ./index.js
>  @ multi ./index.js
> 
> ERROR in ./node_modules/solid-panes/index.js
> Module not found: Error: Can't resolve './profile/profilePane' in '<workspace>/mashlib/node_modules/solid-panes'
>  @ ./node_modules/solid-panes/index.js 132:9-41
>  @ ./index.js
>  @ multi ./index.js
> 
> ERROR in ./node_modules/solid-panes/index.js
> Module not found: Error: Can't resolve './scratchpad/paneWrapper' in '<workspace>/inrupt/mashlib/node_modules/solid-panes'
>  @ ./node_modules/solid-panes/index.js 54:9-44
>  @ ./index.js
>  @ multi ./index.js
> 
> ERROR in ./node_modules/solid-panes/index.js
> Module not found: Error: Can't resolve './trustedApplications/trustedApplicationsPane' in '<workspace>/mashlib/node_modules/solid-panes'
>  @ ./node_modules/solid-panes/index.js 133:9-65
>  @ ./index.js
>  @ multi ./index.js

In other words, the `require` calls to TypeScript aren't bundled. (Nor are those to regular files - presumably, mashlib's Webpack config also compiles solid-panes.)

--- a/.npmignore
+++ b/.npmignore
@@ -1,0 +1,4 @@
+# This file primarily exists to prevent ./dist from being ignored when publishing.
+# (Because it's listed in .gitignore).
+
+coverage

--- a/.travis.yml
+++ b/.travis.yml
@@ -5,3 +5,15 @@ script:
   - npm test
   # Make sure it still builds successfully - the build isn't actually used yet:
   - npm run build
+before_deploy:
+  - node ./travis/bumpversion.js
+deploy:
+  provider: npm
+  # Do not throw away the updated package.json we generated in `before_deploy`:
+  skip_cleanup: true
+  email: "$NPM_EMAIL"
+  api_key: "$NPM_TOKEN"
+  # Note: do not deploy on pull request, because $TRAVIS_BRANCH will be the target branch.
+  tag: "$TRAVIS_BRANCH"
+  on:
+    all_branches: true

--- a/travis/bumpversion.js
+++ b/travis/bumpversion.js
@@ -1,0 +1,19 @@
+/**
+ * npm does not allow you to publish a package with the same version multiple times.
+ * Thus, to publish prerelease versions tagged to the branch they're built from,
+ * we need to generate unique version numbers that are also higher than versions already published.
+ * To achieve this, we append `build<build_number>` to the version number.
+ * The actual release version can eventually be published without the suffix.
+ */
+
+if (!process.env.TRAVIS_BUILD_NUMBER || process.env.TRAVIS_BUILD_NUMBER.length === 0) {
+  console.error('Could not read the build number to bump the package version - aborting publish.')
+  process.exit(1)
+}
+
+const fs = require('fs')
+const path = require('path')
+
+const packageJson = require('../package.json')
+packageJson.version = `${packageJson.version}build${process.env.TRAVIS_BUILD_NUMBER}`
+fs.writeFileSync(path.resolve(__dirname, '../package.json'), JSON.stringify(packageJson))

